### PR TITLE
[MBL-1447] Create fragments for PPO

### PR DESCRIPTION
**Pull request by ifosli**
# 📲 What

Create fragments for PPO, so that we can fetch exactly the data we need for the cards. These fragments can be updated to include more fields if we discover we need them.

# 👀 See

[Jira](https://kickstarter.atlassian.net/browse/MBL-1447)

# ✅ Acceptance criteria

- [x] Project builds (I also mocked together an entire query using these fragments and it worked!)

### CODE REVIEW

File: KsApi/graphql-schema.json
Comment (ifosli) [inline KsApi/graphql-schema.json:1]: Fyi, this giant file is automatically generated (by a script that we run in terminal). I don't typically look through changes here (there's a lot of lines of code) unless something in our build breaks.

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: KsApi/fragments/PPOProjectFragment.graphql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+fragment PPOProjectFragment on Project {
+  creator {
+    email
+    id
+    name
+  }
+  image {
+    id
+    url(width: 1024)
```
Comment (scottkicks) [inline KsApi/fragments/PPOProjectFragment.graphql:11]: Interesting! How does this work?

**Review by scottkicks — APPROVED** [review]

File: KsApi/fragments/PPOProjectFragment.graphql (hunk 1 above)
Comment (ifosli) [reply KsApi/fragments/PPOProjectFragment.graphql:11]: Yeah! I think hardcoded parameters pretty much work the same as the ones we pass in. This one is just copied from our normal project fragment, though

**Pull request merged**